packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-configuration-item.vue

```diff
@@ -27,7 +27,7 @@ import { ref } from 'vue';
 import { useConfirm } from 'primevue/useconfirm';
 import { deleteModelConfiguration } from '@/services/model-configurations';
 
-const emit = defineEmits(['delete', 'use', 'download']);
+const emit = defineEmits(['delete', 'use', 'downloadArchive', 'downloadModel']);
```

Comment:
Why not doing the download here, you already have the configuration?
I do not see the need for extra emit here.

Reply:
Mostly just to match the style of whats going on in here - there are 3 buttons that all use `emit`, it would have felt weird to add a fourth and make it work differently.